src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -239,7 +239,12 @@ void InferRequest::SetBlob(const std::string& name, const Blob::Ptr& data) {
                                            desc.getPrecision().size(),
                                            std::multiplies<size_t>());
 
-    if (dataBinSize != netReqBinSize && !compoundBlobPassed) {
+    bool preProcResize = false;
+    if (is_input) {
+        preProcResize = foundInput->getPreProcess().getResizeAlgorithm() != ResizeAlgorithm::NO_RESIZE;
```

Comment:
Is it only about resize?
Will it work in case if color conversion is required without resizing?

Reply:
Probably not, fixed